src/anomalib/models/image/chatgpt_vision/readme.md

```diff
@@ -0,0 +1,95 @@
+# OpenAI VLM: Zero-/Few-Shot Anomaly Classification
+
+This repository contains the implementation of the `OpenAI VLM`, a model designed for zero-shot and few-shot anomaly detection using OpenAI's GPT-4 for image analysis.
```

Comment:
Is `OpenAI VLM` name of the implementation? It might be confusing to the reader to have multiple naming